### DISCUSSION TITLE
Telegram bot for Firefly III

### DISCUSSION THREAD

**Discussion by cyxou** [firefly-iii/firefly-iii#5521] [General]
Hi!

Thank you for the great Firefly III product 🙌 I've been using it for about a year and really enjoying it 😃

The only thing that was annoying to me is creating new transactions on the go and although there is an Android app out there that facilitates creating transactions but it'is kind of slow to me and requires too many taps/click in order to create a simple transaction like `Coffee $10`. So I decided to create a telegram bot for faster transaction creation and some other stuff. And yes, I am aware that there is already a [telegram bot existing](https://github.com/vjFaLk/firefly-bot) but it's lacking some functionality I need and also I am not that experienced with python in order to contribute to that project. So I decided to create [my own Telegram bot for Firefly III](https://github.com/cyxou/firefly-iii-telegram-bot) which can potentially replace mobile app.

Would appreciate any feedback/contributions and a link to it from the official documentation so other users know about it.

Thanks again for the great project! ❤️
Keep it up!

**Comment by cyxou** [comment]
![image](https://user-images.githubusercontent.com/5896622/148939630-1d7f6c9b-3c6e-42b6-a60c-c833426ea4d7.png)

**Comment by MacPaille** [comment]
Seems a really good idea!!! I also find using the mobile app or even FFIII in my phone browser sometimes cumbersome. I'll give it a try

**Comment by JC5** [comment]
Very cool!

I've added your bot to [this page](https://docs.firefly-iii.org/firefly-iii/other-pages/3rdparty/). It may be some time before it appears, because there is a cache in between.

Thank you for building it, it's much appreciated!
